app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,101 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* universal */
+$gray-light : #777;
+$gray :#555;
+$dark-gray : #222;
+
+body {
+    padding-top: 60px;
```

Comment:
Normally we would use dynamic spacing like `em` or `%` because `px` isnt responsive